### ISSUE TITLE
ES module package @reactioncommerce/api-plugin-carts fails to show exports analysis

### ISSUE THREAD

**Issue by aldeed** [pastelsky/bundlephobia#340]
## Bug
[This package](https://bundlephobia.com/result?p=@reactioncommerce/api-plugin-carts@1.0.0) does not show exports analysis, but the [package.json](https://github.com/reactioncommerce/api-plugin-carts/blob/trunk/package.json) has `type: "module"` and `sideEffects: false`.

Note that this is a Node-only package so it doesn't have a `module` field in `package.json`. Maybe that is why? Nevertheless it is exporting an ES module, and Node doesn't support the `module` field so there is no reason to add it.

**Comment by pastelsky** [comment]
You're right. We don't recognize `type: "module"` today, even though we should. I'd be adding support for this soon.
